impeller/aiks/canvas.cc

```diff
@@ -336,7 +336,7 @@ void Canvas::SaveLayer(Paint paint,
 void Canvas::DrawTextFrame(TextFrame text_frame, Point position, Paint paint) {
   auto lazy_glyph_atlas = GetCurrentPass().GetLazyGlyphAtlas();
 
-  lazy_glyph_atlas->AddTextFrame(std::move(text_frame));
+  lazy_glyph_atlas->AddTextFrame(text_frame);
 
   auto text_contents = std::make_shared<TextContents>();
   text_contents->SetTextFrame(std::move(text_frame));
```

Comment:
How is this not a bug?  This manifests as a bug right?

Reply:
@chinmaygarde may know. I don't understand how this ever worked :)